source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Generating Pdf in Ruby on Rails
```

Comment:
title tag character length is less it should be -55 to 60

Reply:
Done

Reply:
fine